### PR TITLE
Text edits part 2

### PR DESCRIPTION
**Pull request by thesyaf**
added in new lines

### CODE REVIEW

File: StoryofJames.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 It Was the Worst of Times;
 It Was the Worst of Times;
```
Comment (thesyaf) [inline StoryofJames.html:2]: Is this duplicated?

File: StoryofJames.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 
-I was born on a wave of bananas of a full moon,
+I was born on a wave of bananas of a full moon.
 Unlike her, the sea was flat and there wasn't a wave in sight.
-
-Who knew that the world would meet such a lovely man.
+The bananas settled against each other like a finely stitched peice of embroidery.
```
Comment (Jablooo) [inline StoryofJames.html]: maybe the bananas should be the watermelons

File: StoryofJames.html [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
+Tempting me to lie on them like a giant hammock.
```
Comment (thesyaf) [inline StoryofJames.html]: review pls

**Review by thesyaf — COMMENTED** [review]
watermelons added

**Pull request merged**